src/utilities/create-coin.ts

```diff
@@ -0,0 +1,18 @@
+interface CreateCoin {
+  (
+    coin: { id: string; symbol: string },
+    balance: number,
+    onWatchlist: boolean
+  ): object;
+}
+
+export const createCoin: CreateCoin = (
+  coin,
+  balance = 0,
+  onWatchlist = false
+) => ({
+  id: coin?.id,
+  symbol: coin?.symbol,
```

Comment:
don't get why you have `?.` here, as the `coin` is not optional in the interface

Reply:
Yes, you are right. It came from the time where I thought .? everywhere was a good idea. Fixed it.